ntoskrnl/inbv/inbv.c

```diff
@@ -762,6 +782,116 @@ NtDisplayString(IN PUNICODE_STRING DisplayString)
     return STATUS_SUCCESS;
 }
 
+#ifdef INBV_ROTBAR_IMPLEMENTED
+static
+VOID
+NTAPI
+InbvRotationThread(
+    _In_ PVOID Context)
+{
+    ULONG X, Y, Index, Total;
+    LARGE_INTEGER Delay = {{0}};
+
+    // FIXME: This was added to suppress thread hanging
+    KeSetPriorityThread(KeGetCurrentThread(), HIGH_PRIORITY);
```

Comment:
I'd still like to get rid of this before merging

Reply:
@binarymaster, could you please check whether things work without the priority setting? I think they should.

Reply:
Ok, I retested without this change, and found no difference. So this can be removed.